src/mlops_end_to_end_project/calories_burned_model.py

```
@@ -0,0 +1,64 @@
+from typing import Any, Tuple
+
+import mlflow
```

Ruff (0.8.0)

`mlflow` imported but unused

Remove unused import: `mlflow`

(F401)